### PR TITLE
New release info screen - UI (EXPOSUREAP-4591)

### PR DESCRIPTION
**Pull request by SamuraiKek**
This PR adds the UI for the new release info screen. Logic will be implemented in a different PR.

### CODE REVIEW

**Review by d4rken — APPROVED** [review]
lgtm

**Comment by d4rken** [comment]
If the "info" items always look the same, maybe in a future versions we could use a recyclerview to create the entries dynamically?

**Comment by SamuraiKek** [comment]
> If the "info" items always look the same, maybe in a future versions we could use a recyclerview to create the entries dynamically?

Sound like a good idea. I guess we'll see with release 1.13.

File: Corona-Warn-App/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -257,6 +257,12 @@
         <item name="android:textColor">@color/colorTextPrimary1</item>
     </style>
 
+    <style name="subtitleBoldSixteen" parent="@style/TextAppearance.MaterialComponents.Subtitle1">
```
Comment (harambasicluka) [inline Corona-Warn-App/src/main/res/values/styles.xml:260]: Could you please clarify with the designers if we should introduce a new typographie?

Comment (harambasicluka) [reply Corona-Warn-App/src/main/res/values/styles.xml:260]: A new text style is wanted.

**Review by harambasicluka — APPROVED** [review]

File: Corona-Warn-App/src/main/res/values/styles.xml (hunk 1 above)
Comment (SamuraiKek) [reply Corona-Warn-App/src/main/res/values/styles.xml:260]: Talked to the designers and it's indeed a new text style.

**Pull request merged**